django_celery_beat/decorators.py

```diff
@@ -0,0 +1,104 @@
+"""Task decorators."""
+import functools
+
+from celery import Celery
+from celery.app import app_or_default
+
+__all__ = ["periodic_task"]
+
+_app: Celery = app_or_default()
```

Comment:
This line ended up being an issue in my migration testing. Apparently in 2.0.0... 

> - Dropped support for Django < 2.2 and Python < 3.5

We're still on Python 3.5 for library reasons, but this variable annotation isn't supported until Python 3.6 according to [PEP-526](https://www.python.org/dev/peps/pep-0526/). 

I'm going attempt to change this in my own fork to test if a downgrade to 1.6.x and a change to the use of this variable in this module takes care of our upgrade path problem.

Reply:
I've cherry-picked these changes onto 1.6.0, but I haven't yet had luck starting a `celery worker` that picks up `@periodic_task` tasks in startup.

I copied the technique to get the app from the scheduler here:
https://github.com/bevy/django-celery-beat/blob/a8b0dec3e04f1b4fa434323541e32719d560f585/django_celery_beat/schedulers.py#L59

I'll try `app_or_default` instead and see if that bears fruit.